merino-web/src/extractors.rs

```diff
@@ -65,17 +69,23 @@ impl FromRequest for SuggestionRequestWrapper {
                 dma: location.dma,
                 city: location.city,
                 device_info,
+                client_variants,
             }))
         }
         .boxed_local()
     }
 }
 
 /// A query passed to the API.
+#[serde_as]
 #[derive(Debug, Deserialize)]
 struct SuggestQuery {
     /// The query to generate suggestions for.
     q: String,
+    /// The client variants to generate suggestions with.
+    #[serde_as(as = "Option<StringWithSeparator::<CommaSeparator, String>>")]
+    #[serde(default)]
+    client_variants: Option<Vec<String>>,
```

Comment:
As a future consideration, it might be worth figuring out how to make this just a `Vec<String>`, instead of making an `Option`. I don't think at this level there is a semantic difference between no client variants and an empty list.